test/fixtures/spread-operator.js

```diff
@@ -0,0 +1,23 @@
+const Home = () => (
```

Comment:
can you remove this fixture file and add a few cases to this https://github.com/zeit/styled-jsx/blob/803896fc36f81d6689ee0f074306fc130a731698/test/fixtures/attribute-generation-classname-rewriting.js ?

```jsx
<div {...{ id: 'foo' }} />
<div {...{ className: 'foo' }} />
<div {...{ className: 'foo' }} className="test" />
<div className="test" {...{ className: 'foo' }} />
<div  {...{ className: 'foo' }} {...bar} />
<div  {...{ className: 'foo' }} {...bar} className="test" />
<div className="test"  {...{ className: 'foo' }} {...bar} />
// more corner cases???
```